packages/oasgraph-cli/README.md

```diff
@@ -19,29 +19,49 @@ npm i -g oasgraph-cli
 
 
 ## Usage
-The basic usage of the CLI takes the specified OAS, create a GraphQL interface for it, and start a server to host the GraphQL interface.
 
-You can specify a local file containing the OAS specification or a remote url such as `http://127.0.0.1:3000/openapi.json`. Additionally, you can specify an optional port number so you can have multiple GraphQL servers for testing in the same machine.
+```
+Usage: oasgraph <OAS JSON file path or remote url> [options]
+
+Options:
+  -V, --version             output the version number
+  -p, --port <port>         select the port where the server will start
+  -s, --strict              create a GraphQL schema matching the given OAS perfectly
+  -a, --addSubOperations    nest operations based on path hierarchy
+  -f, --fillEmptyResponses  wrap specific operations with no response schema
```

Comment:
Wrap with what? Also, maybe instead of "specific", we could point to the 204 status code, which I believe is a mandatory precondition?

Reply:
Wrap with GraphQL? Maybe `run OASGraph on legitimate operations with no response schema, such as those with HTTP 204 code, rather than ignore them`?

Yes, "specific" refers to 204 but I wasn't sure if we wanted to expand the preconditions.

Reply:
`create placeholder schema for operations with HTTP status code 204 (no response) rather than ignore them`?